### PR TITLE
rdfparse + webpack, fixes #3 and makes better dev environment for npm packages

### LINKED ISSUE #3: Use Comunica to fetch RDF
Currently, N3.js is used to parse RDF and a fetch to get the RDF.
However, this doesn't work for embedded JSON-LD as seen here: https://dylanvanassche.be/#me

### PR DESCRIPTION
**Pull request by bjdmeest**

### CODE REVIEW

**Comment by pheyvaer** [comment]
Hi @bjdmeest

Thanks for these changes! I see that when trying it locally that the participants are not loaded anymore. Does this still work on your side?

**Comment by bjdmeest** [comment]
hmmm, yes, I tried with the dummy peoples and our calendar links.. there's no console error?

**Comment by pheyvaer** [comment]
No, I see a request to get the participants is made. And some processing is a happening, but I think it's not using the data from the request.

**Comment by bjdmeest** [comment]
I have below, I have the feeling the right data fetched and processed?

image

**Comment by pheyvaer** [comment]
What you have is indeed correct. But I see 

image

**Comment by pheyvaer** [comment]
Also, I see that I get the logs from the minified index.js, but you seem to have the full version of it. Are you running it not with `npm start`?

**Comment by bjdmeest** [comment]
Those results are using the development config in webpack.config.js, but the results/console.log are the same, just better links to the actual source code :)

**Comment by bjdmeest** [comment]
I added a `npm watch` script, that runs a dev environment, next to the `npm start` script that runs a production environment

File: README.md [hunk 1]
```diff
@@ -7,20 +7,23 @@ by using their availability calendar which is made available through a Solid pod
 ## Run locally via HTTP server
 
 1. Install dependencies via `npm i`.
-2. Run HTTP server via `npm start`.
+2. Run production HTTP server via `npm start` (this first runs webpack to generate the browser index.js file), OR
+3. Run development HTTP server (that reloads when you make changes) via `npm watch`
```
Comment (pheyvaer) [inline README.md:11]: Should be `npm run watch` I guess?

File: js/utils.js [hunk 2]
```diff
@@ -104,7 +98,7 @@ function getSelectedParticipantUrls(participants) {
   return urls;
 }
 
-async function fetchParticipantWebIDs(fetch, employeesUrl) {
+export async function fetchParticipantWebIDs(fetch, employeesUrl) {
```
Comment (pheyvaer) [inline js/utils.js:101]: I fixed the issue on my side by passing the parameter `participants` do this function. I guess that with making it webpack it doesn't these global variables anymore.

**Comment by pheyvaer** [comment]
Nice! When I try it your WebID doesn't show. Is that the same on your side? I thought your changes fixed some things so that yours would show up.

**Comment by pheyvaer** [comment]
Are we in that case maybe dealing with an issue related to CSS? @joachimvh any clues from your side?

**Comment by joachimvh** [comment]
Would need some more context as to the full setup, but sounds similar to the CORS issues you had no? Depends a bit on who is throwing what CORS errors when trying to contact what.

File: js/utils.js [hunk 3]
```diff
@@ -29,7 +39,7 @@ function getParticipantViaCalendarUrl(url, participants) {
   return null;
 }
 
-function getRDFasJson(url, frame, fetch) {
+export async function getRDFasJson(url, frame, fetch) {
```
Comment (pheyvaer) [inline js/utils.js:42]: Why is this function async?

File: README.md [hunk 4]
```diff
@@ -7,20 +7,23 @@ by using their availability calendar which is made available through a Solid pod
...
 2. Update the method `fetchParticipantWebIDs` in `js/utils.js` to extract the people from the data that is found at `employeesUrl`.
 3. Update the variable `participants` in `js/index.js` to remove the dummy people.
+4. Update the generated browser index.js file via `npx webpack`.
 
 ## Development
 
 - You find all HTML of the application in `index.html`.
 - You find the frequently asked questions page in the folder `faq`.
-- You find all used libraries in the folder `lib`.
+- You find some used libraries in the folder `lib`, most are handled using webpack. To create a development webpack bundle, use `webpack.dev.config.js`
```
Comment (pheyvaer) [inline README.md:24]: Missing dot at the end of sentence.

File: js/utils.js (hunk 3 above)
Comment (bjdmeest) [reply js/utils.js]: woops. fixed

**Comment by bjdmeest** [comment]
nevermind, found it: the fetch function from solid-authn-js doesn't work well with redirect cors. Replacing 'fetching a webID' with the default fetch did the trick, PR should be complete now!

**Pull request merged**